pkgs/misc/cups/drivers/brlaser/default.nix

```diff
@@ -14,7 +14,12 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ];
   buildInputs = [ zlib cups ];
 
-  cmakeFlags = [ "-DCUPS_SERVER_BIN=lib/cups" "-DCUPS_DATA_DIR=share/cups" ];
```

Comment:
You can use `cmakeFlags = [ "-DCUPS_SERVER_BIN=${placeholder "out"}/lib/cups" "-DCUPS_DATA_DIR=${placeholder "out"}/share/cups" ];

Reply:
cool, didn't know that! Fixed.